components/Pagination.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Link from 'next/link';
+
+interface PaginationProps {
+  total: number;
+  current: number;
+  pageSize: number;
+}
+
+const Pagination = ({ total, current, pageSize }: PaginationProps) => {
+  const numberOfPage = Math.round(total / pageSize);
+
+  return (
+    <nav>
+      <ul className="list-reset flex justify-center mt-6">
+        {current > 1 && (
+          <li key="first" className="inline-block flex-none mx-1">
+            <Link href="?page=1">
+              <a className="no-underline text-primary hover:text-black">
+                « First
+              </a>
+            </Link>
+          </li>
+        )}
+        {current > 1 && (
+          <li key="prev" className="inline-block flex-none mx-1">
+            <Link href={`?page=${current - 1}`}>
```

Comment:
Click on page 2 seems not working:

![image](https://user-images.githubusercontent.com/12707960/104794798-a655e200-57dc-11eb-902c-5ffb88a9c474.png)

See the warning
![image](https://user-images.githubusercontent.com/12707960/104794822-c4bbdd80-57dc-11eb-8cda-b68a16a212f9.png)

So I think we have to provide a prefix prop for the URL so it can be `${prefixUrl}?page=${current}`. we can get the prefix on the parent component via `useRouter`.